resources/context_widget.ui

```diff
@@ -21,17 +21,42 @@
     <number>0</number>
    </property>
    <item>
-    <widget class="QLabel" name="header">
-     <property name="sizePolicy">
-      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
-       <horstretch>0</horstretch>
-       <verstretch>0</verstretch>
-      </sizepolicy>
+    <widget class="QFrame" name="frame">
```

Comment:
Added a QFrame in the Vertical layout. Added a Horizontal Layout underneath the QFrame. Then,  Added the context QLabel inside the Horizontal Layout and the checkbox next to it.